docs/UserGuide.md

```diff
@@ -349,12 +367,26 @@ Put alerts here, if any
 <div markdown="block" class="alert alert-info">
 
 **Note:**
-Put notes here
+``find-a`` is case-insensitive and does not require the whole word to match, for example, ``find-a la`` will match
+with "CS2103T Lab"
+
 </div>
 
-**Examples:** show some examples with the parameters
+**Examples:** 
+- ``find-a CS2103T`` to find all assignments with "CS2103T" in the assignment name
+- ``find-a lab`` to find all assignments with the word "lab" in the assignment name
 
-**When you might use it:** give some use cases
+**When you might use it:**
```

Comment:
Small thing but can standardise the find-a in the use case should be `find-a`. Also recommend shortening each use case